src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Random random = new Random();
+        int randomIndex = random.nextInt(Color.values().length);
+
+        return  Color.values()[randomIndex].toString();
```

Comment:
Use the `name()` method instead of `toString()` for getting the String representation of enum constants to avoid issues if the `toString()` method is overridden.